mirgecom/initializers.py

```diff
@@ -8,6 +8,7 @@
 .. autoclass:: Lump
 .. autoclass:: Uniform
 .. autoclass:: AcousticPulse
+.. autoclass:: MultiLump
```

Comment:
It might be helpful to rename  Lump-->MassLump, and MultiLump-->MulticomponentLump. The "Lump" designation is only there because they both initialize solution components using a Gaussian.

Reply:
1ba6198
84f3a40